pages/persoonsgegevens/[person].tsx

```diff
@@ -111,6 +112,11 @@ export default function MultistepForm1() {
     }
   }, [huwelijkId, initializeMarriage, marriageOptions, reservation, setMarriageOptions]);
 
+  useEffect(() => {
+    if (!formState.isSubmitted) return;
+    reset(undefined, { keepValues: true, keepErrors: true, keepIsValid: true, keepIsSubmitted: !formState.isDirty });
```

Comment:
Een comment voor deze `useEffect` zou geen overbodige luxe zijn 😅

Reply:
Fair enough! Ik voeg een comment toe for now, zal ook nog kijken of er een meer self-explanatory oplossing is want dit gaat vervelend zijn om op iedere form page met gelijknamige logica te zetten.